packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -63,9 +107,11 @@ def test_ip_per_container(cluster):
         ensure_routable(cmd, service_points)()
 
 
+@skip_if_lb_disabled
 def test_ip_per_container_with_named_vip(cluster):
     """Test if we are able to connect to a task with ip-per-container mode using named vip
     """
+    test_if_overlay_ok(cluster)
```

Comment:
So, right now, test_if_overlay_ok acts an implicit barrier to the tests that might rely on the overlay. Sometimes the overlay takes some time to come up. If someone reorders the tests, it could be bad. I don't really want this to be an independent fixture either.